packages/back-end/src/services/stats.ts

```diff
@@ -218,6 +201,64 @@ print(json.dumps({
   }
 }
 
+function createStatsEngineData(
+  params: ExperimentMetricAnalysisParams
+): DataForStatsEngine {
+  const {
+    variations,
+    metrics,
+    phaseLengthHours,
+    coverage,
+    analyses,
+    queryResults,
+  } = params;
+
+  const phaseLengthDays = Number(phaseLengthHours / 24);
+
+  return {
+    metrics: metrics,
+    query_results: queryResults,
+    analyses: analyses.map((a) =>
+      getAnalysisSettingsForStatsEngine(
+        a,
+        variations,
+        coverage,
+        phaseLengthDays
+      )
+    ),
+  };
+}
+
+export async function analyzeSingleExperiment(
+  params: ExperimentMetricAnalysisParams
+): Promise<ExperimentMetricAnalysis> {
+  const result = (
+    await runStatsEngine([{ id: "_", data: createStatsEngineData(params) }])
+  )?.[0];
+
+  if (!result) {
+    throw new Error(
+      "Failed to analyze manual snapshot; Stats Engine returned no results"
+    );
+  }
+  return result.results;
+}
+
+export async function analyzeMultipleExperiments(
+  experimentParams: {
+    experiment: string;
+    params: ExperimentMetricAnalysisParams;
```

Comment:
perhaps better to put `experimentId` into `ExperimentMetricAnalysisParams`

Reply:
Done, ty.